MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of week number text.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfSchedulerNormalHeaderFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+             Size of header text in scheduler.                 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfSchedulerNormalViewHeaderDayTextFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of view header day text.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfSchedulerNormalViewHeaderDateTextFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of view header date text.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+</table>
+
+## SfTreeMap
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+
+    <tr>
+        <td rowspan="18">
+            SfTreeMapStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfTreeMapTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, it is possible to customize the appearance of the SfTreeMap without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfTreeMapTheme">CommonTheme</x:String> 
+                <Color x:Key="SfTreeMapNormalBackground">Blue</Color> 
+                <Color x:Key="SfTreeMapGroupItemNormalBackground">Green</Color> 
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+	</tr>
+    
+    <tr>
+    <td>
+            SfTreeMapNormalBackground    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of treemap.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+    <tr>
+    <td>
+            SfTreeMapSelectedItemStroke     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of treemap item when it is selected.
```

Comment:
color of a treemap item's border when it is selected.

Reply:
Updated